src/main/java/com/minecolonies/util/SchematicWrapper.java

```diff
@@ -379,7 +379,7 @@ private static Schematic scanSchematic(World world, BlockPos from, BlockPos to)
                     }
 
                     TileEntity tileEntity = world.getTileEntity(worldPos);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'tileEntity' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)